backend/app/Actions/Pages/GetPageViewsResponse.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Actions\Pages;
+
+final class GetPageViewsResponse
+{
+    private $previews;
+
+    public function __construct($previews)
+    {
+        $this->previews = $previews;
```

Comment:
"page view" and "preview" are completely different, in context our platform "page view" is a one visit of tracked web site